src/dcc/ProgrammingTrackBackend.hxx

```diff
@@ -200,9 +204,11 @@ public:
                 return call_immediately(STATE(exit_service_mode));
 
             case ProgrammingTrackRequest::Type::SEND_RESET:
+                if (has_short) request()->hasShortCircuit_ = 1;
```

Comment:
Can we add braces in stead of having single line if statements? Same in the next case statement.

Reply:
done